### PR TITLE
Feature/start end sprint date

### PR DESCRIPTION
**Pull request by danylokarpenko**

### CODE REVIEW

File: client/src/components/common/SprintModal/CreateSprintModal/index.tsx [hunk 1]
```diff
@@ -1,11 +1,15 @@
 import React, { useState } from 'react';
+import DatePicker from 'react-datepicker';
```
Comment (timagixe) [inline client/src/components/common/SprintModal/CreateSprintModal/index.tsx:2]: Let's move date picking from CreateSprintModal to EditSprintModal. Additionally, when editing sprint, let's make date picking disabled when toggle `isActive` is set to false.

File: client/src/components/common/SprintModal/CreateSprintModal/index.tsx [hunk 2]
```diff
@@ -82,6 +123,54 @@ const CreateSprintModal = (props: Props) => {
...
+									name="StartTime"
+									dateFormat="MM/dd/yyyy h:mm aa"
+									showTimeSelect
+									selected={startDate}
+									onChange={handleStartDatePick}
+								/>
+								{!isDateValid && (
+									<span style={{ color: 'red' }}>
+										The start date of a sprint must be before the end date.
+									</span>
```
Comment (timagixe) [inline client/src/components/common/SprintModal/CreateSprintModal/index.tsx:154]: Error is showed when start date is picked manually and is still before end date. Link to trello - [click me](https://trello.com/c/jZYlZHjG/263-add-start-and-end-date-for-the-sprint#comment-5f48ad0a6798f274e05f203e)

File: client/src/components/common/SprintModal/CreateSprintModal/helpers.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export const getNextDate = (duration: number | 'custom', startDate: Date) => {
+	const nextDate = new Date(
+		startDate.getFullYear(),
+		startDate.getMonth(),
+		startDate.getDate() + 7 * (duration === 'custom' ? 1 : duration),
+		startDate.getHours(),
+		startDate.getMinutes() + 15,
+	);
+	return nextDate;
+};
```
Comment (timagixe) [inline client/src/components/common/SprintModal/CreateSprintModal/helpers.ts:10]: What if startDate is not Date? Let's handle this behavior too and avoid crashing. [Link](https://trello.com/c/jZYlZHjG/263-add-start-and-end-date-for-the-sprint#action-5f48adb8e6dffc0efb3de3de).

**Review by timagixe — CHANGES_REQUESTED** [review]

**Comment by danylokarpenko** [comment]
image
Added localization

**Review by timagixe — APPROVED** [review]

**Pull request merged**